### ISSUE TITLE
How to compute the error rate?

### ISSUE THREAD

**Issue by weiwei-ww** [sharathadavanne/multichannel-sed-crnn#2]
Hi, I got a little confused by how you compute the error rate in metrics.py. As introduced in Mesaros's paper, is it supposed to calculate the substitution rate, deletion rate and insertion rate for each frame first and then compute the error rate based on them? I also checked with the example from the Figure 4 in that paper, which should give an error rate of 0.5. Your function er_overall_framewise() is giving an error rate of 0.4.

The paper can be found here: http://www.mdpi.com/2076-3417/6/6/162/pdf

I used the following codes for testing:
`O = np.array([[1, 1, 0, 1, 1], [1, 0, 0, 0, 1], [1, 0, 0, 0, 1]])`
`T = np.array([[1, 0, 1, 1, 0], [1, 0, 0, 1, 1], [1, 1, 0, 1, 1]])`
`error_rate = er_overall_framewise(O, T)`
`print(error_rate)`

The result should be 0.5 instead of 0.4 according to that paper.

**Comment by sharathadavanne** [comment]
Hi @weiwei-ww for your input the error rate should actually be 0.4. According to the equations 4 and 5 in the paper you referred, the intermediate statistics are N=10, S=2, D=2, and I=0, resulting in an ER of 4/10 = 0.4.

**Comment by weiwei-ww** [comment]
Hi @sharathadavanne, thanks for you answer. As for the example from that paper, here is how i computed the S value. There are three frames, so I first calculated the number of FN and FP for each frame. The result of FN is [1, 1, 2]. The result of FP is [2, 0, 0]. Then we follow the formula in the paper (find the min of FN and FP) to calculate the S value for each frame and the result is [1, 0, 0]. Finally I added these three numbers together to get the final value of S = 1. Similarly, I can compute the D = 3 and I =1. This is why my error rate is 0.5 instead of 0.4. May I know which part of my calculation is different from yours?
